workflow-rx2/src/main/java/com/squareup/workflow/rx2/TestUtils.kt

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Let's make a new module called `workflow-test` and put this there, so it doesn't pollute non-test code. I think we will probably end up putting more stuff in there at some point.

@rjrjr WDYT?

Reply:
Yes yes yes.